### PR TITLE
Add Ulimits capability to aws task creation

### PR DESCRIPTION
**Pull request by bachiri**
### Description : 
Add Ulimits Capability to the plugin to configure the desired Ulimit: 

image

### Github Issue: 
https://github.com/jenkinsci/amazon-ecs-plugin/issues/257

### How the changes were tested: 
The plugin was tested in this [Jenkins instance](http://54.146.59.68:8080/) by building the .hpi file and also internally. 

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by bachiri** [comment]
Hi @Stericson if can you have a look at this PR. Thanks in advance.

**Review by Stericson — CHANGES_REQUESTED** [review]

**Comment by Stericson** [comment]
@bachiri Would you mind addressing the small items I noted? Following that, I'll get this merged.

File: src/main/java/com/cloudbees/jenkins/plugins/amazonecs/ECSTaskTemplate.java [hunk 1]
```diff
@@ -1303,6 +1330,44 @@ public String getDisplayName() {
...
+        @DataBoundConstructor
+        public UlimitEntry(Integer softLimit,Integer hardLimit, String ulimitName) {
+            this.softLimit = softLimit;
+            this.hardLimit = hardLimit;
+            this.ulimitName = ulimitName;
+        }
+
+        @Override
+        public String toString() {
+            return "PortMappingEntry{" +
```
[the remark this answers is not in the document]
Comment (bachiri) [reply src/main/java/com/cloudbees/jenkins/plugins/amazonecs/ECSTaskTemplate.java]: @Stericson Done for both.

**Comment by bachiri** [comment]
Thanks @Stericson feedbacks addressed.

**Review by Stericson — APPROVED** [review]

**Comment by bachiri** [comment]
Thanks @Stericson for the quick feedback/reactivity, when should I expect a release containing this feature.

**Comment by Stericson** [comment]
@bachiri I'll work to get it out in the next couple of weeks.

**Comment by bachiri** [comment]
> @bachiri I'll work to get it out in the next couple of weeks.

Thanks @Stericson 🙏  Looking forward for the release to update our internal tooling and mitigate the issue we are currently facing.

**Pull request merged**